### PR TITLE
[#172473189] Fixed missing https on FAQ external link

### PR DESCRIPTION
**Pull request by CrisTofani**
**Short description:**
This PR fixes crash caused by broken links.

**List of changes proposed in this pull request:**
- locales/en/faq/faq2.md
- locales/en/faq/faq5.md
- locales/en/profile/profile_appinfo_main.md
- locales/it/faq/faq2.md
- locales/it/faq/faq3.md
- locales/it/faq/faq5.md

### CODE REVIEW

File: locales/en/faq/faq5.md [hunk 1]
```diff
@@ -1 +1 @@
-You can connect to the address [https://reservazionicie.interno.gov.it](prenotazionicie.interno.gov.it) and type the name of your municipality, to check whether it uses the CIE Agenda online dating system. If yes, you can proceed to book the appointment directly from the site, without any login. Alternatively, you can always ask for an appointment online on the website of your municipality, or by going to one of the municipal offices.
\ No newline at end of file
+You can connect to the address [https://reservazionicie.interno.gov.it](https://prenotazionicie.interno.gov.it) and type the name of your municipality, to check whether it uses the CIE Agenda online dating system. If yes, you can proceed to book the appointment directly from the site, without any login. Alternatively, you can always ask for an appointment online on the website of your municipality, or by going to one of the municipal offices.
```
Comment (Undermaken) [inline locales/en/faq/faq5.md:1]:
```suggestion
You can connect to the address [https://reservazionicie.interno.gov.it](https://www.prenotazionicie.interno.gov.it) and type the name of your municipality, to check whether it uses the CIE Agenda online dating system. If yes, you can proceed to book the appointment directly from the site, without any login. Alternatively, you can always ask for an appointment online on the website of your municipality, or by going to one of the municipal offices.
```

File: locales/it/faq/faq5.md [hunk 2]
```diff
@@ -1 +1 @@
-Puoi collegarti all’indirizzo [https://www.prenotazionicie.interno.gov.it/](prenotazionicie.interno.gov.it) e digitare il nome del tuo Comune, per verificare se utilizza il sistema di appuntamenti online Agenda CIE. Se sì, puoi procedere alla prenotazione dell'appuntamento direttamente dal sito, senza nessun login. In alternative, puoi chiedere un appuntamento sempre online sul sito del tuo Comune, oppure recandoti ad uno degli uffici comunali.
+Puoi collegarti all’indirizzo [https://www.prenotazionicie.interno.gov.it/](https://prenotazionicie.interno.gov.it) e digitare il nome del tuo Comune, per verificare se utilizza il sistema di appuntamenti online Agenda CIE. Se sì, puoi procedere alla prenotazione dell'appuntamento direttamente dal sito, senza nessun login. In alternative, puoi chiedere un appuntamento sempre online sul sito del tuo Comune, oppure recandoti ad uno degli uffici comunali.
```
Comment (Undermaken) [inline locales/it/faq/faq5.md:1]:
```suggestion
Puoi collegarti all’indirizzo [https://www.prenotazionicie.interno.gov.it/](https://www.prenotazionicie.interno.gov.it) e digitare il nome del tuo Comune, per verificare se utilizza il sistema di appuntamenti online Agenda CIE. Se sì, puoi procedere alla prenotazione dell'appuntamento direttamente dal sito, senza nessun login. In alternative, puoi chiedere un appuntamento sempre online sul sito del tuo Comune, oppure recandoti ad uno degli uffici comunali.
```

File: locales/en/faq/faq5.md (hunk 1 above)
Comment (CrisTofani) [reply locales/en/faq/faq5.md]: Fixed in [efebc9a](https://github.com/pagopa/io-app/pull/1721/commits/efebc9ace7434ea0b5e985a1871bb165dde5062a) 
Also changed the wrongly typed url description `[https://reservazionicie.interno.gov.it]`

File: locales/it/faq/faq5.md (hunk 2 above)
Comment (CrisTofani) [reply locales/it/faq/faq5.md]: Fixed in [efebc9a](https://github.com/pagopa/io-app/pull/1721/commits/efebc9ace7434ea0b5e985a1871bb165dde5062a)

**Pull request merged**